.github/workflows/style_type_checks.yml

```diff
@@ -9,22 +9,16 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - uses: extractions/setup-just@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.8'
+    - uses: actions/setup-python@v4
     - name: Install dependencies
       run: |
         pip install .
-        pip install click black mypy
-        pip install types-python-dateutil
-        pip install types-waitress
-        pip install types-PyYAML
-    - name: Style and type checks
-      run: |
-        just black
-        just mypy
+        # todo: install also `black[jupyter]`
```

Comment:
that is Back's suggesting, and I think it would be good to enable it
but as it was not enabled yet so, I did not want to introduce it without a discussion

Reply:
so once it will be installed it will also lint ipython notebooks